src/main/java/com/example/springlabs/StoreController.java

```diff
@@ -4,21 +4,23 @@
 import com.example.springlabs.model.Product;
 import com.example.springlabs.services.CategoryService;
 import com.example.springlabs.services.ProductService;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 
 import java.util.ArrayList;
+import java.util.LinkedHashSet;
 import java.util.List;
 
 @Controller
+@AllArgsConstructor
 public class StoreController {
 
-    @Autowired
     private ProductService productService;
```

Comment:
now we can make it final, same for `categoryService`

Reply:
I will rollback it since it does not meet the topic of changes and we have Issue #6 and it will be fixed there